package.json

```
@@ -7,7 +7,7 @@
     "build": "rimraf ./build && tsc",
     "lint": "xo",
     "lint:fix": "xo --fix",
-    "start": "npm run build && node build/index.js",
+    "start": "nodemon --config nodemon.json",
```

This will be better I think.

```suggestion
    "start": "npm run build && node build/index.js",
    "start:nodemon": "nodemon --config nodemon.json",
```

---

What do you think of "dev" instead of "start:nodemon"?

ts-node could be invoked directly in the start command (eliminating the need to create build/ and without nodemon's process reloading). I think it's a little nicer to not have to create build/ in it's entirety every time the app is started but it would still be creating files in /tmp under the hood so doesn't make much difference I guess.

---

I've restored the behaviour of "start" and added "dev" as described above, with a short description in README.md. If you strongly prefer "start:nodemon" please let me know and I'll update it.

---

Nope, `dev` is fine with me.